localization/english/wc_game_rules_l_english.yml

```diff
@@ -4,3 +4,11 @@
  setting_wc_lore_characters_generate_families_on_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will be affected by it."
  setting_wc_lore_characters_generate_families_off:0 "Off"
  setting_wc_lore_characters_generate_families_off_desc:0 "If vanilla $rule_generate_families$ game rule is enabled, lore characters will not be affected by it."
+
+ rule_wc_interracial_marriage:0 "#Bold Warcraft#!: Interracial Marriages"
+ setting_wc_interracial_marriage_no_modifier:0 "Unrestricted"
+ setting_wc_interracial_marriage_no_modifier_desc:0 "Interracial marriages will not suffer a negative modifier."
```

Comment:
Previously unrestricted meant any race can make marriage with every race.
I would bring this option back. This part of the code was responsible for it.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/113878065-53cc0680-97ca-11eb-8c31-2ac10edeeb11.png)

</details>

Reply:
> 
> 
> Previously unrestricted meant any race can make marriage with every race.
> I would bring this option back. This part of the code was responsible for it.
> Click to expand
> 
> ![image](https://user-images.githubusercontent.com/46384992/113878065-53cc0680-97ca-11eb-8c31-2ac10edeeb11.png)

Alright, I will make another one, one for no modifier and one for totally unrestricted